cron_scripts/inactive_idle_accounts.pl

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use lib 'lib';
+
+use Getopt::Long;
+use ZCS::API;
+use Data::Dumper;
+use JSON;
+use File::Slurp;
+use POSIX qw(strftime);
+
+my $config_file = 'config/config.json';
+
+my %opts = (
+    last_login_days => 97,
+    new_status      => 'locked'
+);
+
+my $res =
+  GetOptions( \%opts, "help", "debug", "last_login_days=i", "new_status=s" );
+
+if ( $opts{help} ) {
+    print_usage();
+    exit 0;
+}
+elsif ( !$res ) {
+    print_usage();
+    exit 1;
+}
+
+my $config = JSON->new->utf8->relaxed->decode( scalar read_file $config_file);
+die "MAILSTOREHOST is not defined in config file"
+  if ( !defined( $config->{MAILSTOREHOST} ) || $config->{MAILSTOREHOST} eq "" );
+die "MAILSTOREUSER is not defined in config file"
+  if ( !defined( $config->{MAILSTOREUSER} ) || $config->{MAILSTOREUSER} eq "" );
+die "MAILSTOREPWD is not defined in config file"
+  if ( !defined( $config->{MAILSTOREPWD} ) || $config->{MAILSTOREPWD} eq "" );
+
+my $new_status      = $opts{new_status};
```

Comment:
Do we need to do any validation of this arg?  Or just continue and let soap updates fail if an invalid status is specified?